plugins/org.genivi.faracon/src/org/genivi/faracon/franca2ara/SomeipFrancaDeploymentData.xtend

```diff
@@ -0,0 +1,40 @@
+package org.genivi.faracon.franca2ara
+
+import com.google.inject.Singleton
+import java.util.Map
+import org.franca.core.franca.FInterface
+import org.franca.core.franca.FTypeCollection
+import org.franca.deploymodel.core.FDeployedInterface
+import org.franca.deploymodel.core.FDeployedTypeCollection
+import org.genivi.commonapi.someip.Deployment
+
+@Singleton
+class SomeipFrancaDeploymentData {
+	
+	var Map<FInterface, Deployment.InterfacePropertyAccessor> interfacesDeploymentData
+	var Map<FTypeCollection, Deployment.TypeCollectionPropertyAccessor> typeCollectionDeploymentData
+
+	def clear() {

```

Comment:
also here: if we need to clear a lot things up before we can start a transformation, we should think about chaning the injection mechanism in a way that we get new instance instead of refreshing the old ones.

Reply:
This is another preparation step at the beginning of the conversion of a full system. See my comment above regarding this topic.